script/Ciel/Day 12/12_01_CIEL12_3C.txt

```diff
@@ -31,72 +35,81 @@ I rush into the lobby and finally come to a halt.
 [sha:70de143fcfe3632ce028299cbe1394f9751f92b2]{
 -- Page 135, Offset 31215.
 -- ぜいぜいと肩で息をして心臓を落ち着ける。
-I try to calm down my labored breathing and frantic heart.
+Panting heavily, I calm my racing heart.
 }
 [sha:35ac40b38af6158dc69acd3d43017d4f1574e000]{
 -- Page 135, Offset 31216.
 -- 家の中で走りまわるのはマナー違反だし、休憩がてらここからは早歩きに切り替えよう。
-Running inside the mansion would be a breach of etiquette, so I'll take a breather and switch to a brisk walk instead.
+I'd better slow to a brisk walk from here,
+ both because running inside the mansion would be a breach of etiquette,
+ and to give myself a chance to catch my breath.
 }
 [sha:34c0ce4fbea0b2c72e62a3a8851cd5a33fd714a5]{
 -- Page 136, Offset 31217.
 -- 「あら志貴さん。今日はお早いお帰りなんですね」
-"Well, if it isn't Shiki-san. You sure are early today, huh."
+// This page only if you've scared Hisui off I guess
```

Comment:
The trigger for the extra scenes spanning from Day 11-13 (eventually leading to a bad end) is going INTJ mode and helping out Noel kill the vampires on Day 9.

Reply:
Good to know